grow/commands/upload_translations.py

```diff
@@ -23,21 +23,21 @@
                    ' to uploading.')
 @click.option('--extract', '-x', default=True, is_flag=True,
               help='Whether to extract translations prior to uploading.')
-@click.option('--include-obsolete/--no-include-obsolete', default=None,
-            is_flag=True,
-            help='Whether to include obsolete messages. If false, obsolete'
-                 ' messages will be removed from the upload. By'
-                 ' default, Grow cleans obsolete messages from the upload')
+@click.option('--prune-obsolete', default=False, is_flag=True,
```

Comment:
Thanks -- when it comes to the command-line I've found that "less is more" and it's easier to explain when the flags are short and sweet. :)